ipaserver/plugins/host.py

```diff
@@ -252,6 +252,11 @@ def validate_ipaddr(ugettext, ipaddr):
     return None
 
 
+def resolve_fqdn(name):
+    hostentry = api.Command['host_show'](name)['result']
+    return hostentry['fqdn'][0]
+
```

Comment:
It is probably worth it to retain the host validation. It can save an LDAP read if a FQDN is passed in and I don't think it changes the behavior. Or does it?

Reply:
The idea of this is to act as a existence check so the LDAP search will be performed regardless of whether a FQDN is passed or not. I removed it because of this, after all the host search already returns the FQDN 'validated'.

Reply:
Yes of course, that's the whole point of this exercise.